refresh_rate: 300.0
debug_mode  : FALSE

max_lin_speed: 1.0
max_rot_speed: 1.0
control_frame: "world"
max_x_coord  : +1
min_x_coord  : -1
max_y_coord  : +1
min_y_coord  : -1
max_z_coord  : +1
min_z_coord  : -1

